WebApp/autoreduce_webapp/autoreduce_webapp/backends.py

```diff
@@ -8,16 +8,17 @@
 UOWS authentication backend
 """
 import logging
+
+# pylint:disable=imported-auth-user
```

Comment:
https://github.com/PyCQA/pylint-django/issues/244

I read through this a couple of times but I am still not 100% clear what the reason for this warning is

> Django recommends that you do not use the User model directly. Neither to issue queries against it, nor to use it as a foreign key reference. Instead you should be using the get_user_model() function and the AUTH_USER_MODEL setting.
The reason for all of this is that this can be overridden and applications which do not hard-code the values are in theory more portable downstream. That is they can be used directly inside an environment where the User model has been overridden.

Maybe an alternative is to just replace the `User` usage with `get_user_model()`

Reply:
I added this ignore as there is an issue for this .